packages/sol6/src/yang/etsi-nfv-ns.yang

```diff
@@ -1045,7 +1046,7 @@ submodule etsi-nfv-ns {
                information element";
           }
 
-          leaf affinity-type {
+          leaf affinity-or-anti-affinity {
```

Comment:
To be consistent with the rest of the convention, don't you want this to be 'leaf type'? After all, the next attribute is also just 'leaf scope'.

Reply:
Agree, think affinity-type is better.

Reply:
You mean 'type' is better. BTW, IFA011 also calls the attribute 'type'.

Reply:
I would suggest that we be consistent and call it 'type' in both vnf.yang and ns.yang. I will bring it up with IFA014 to make it consistent with IFA011.